src/chai-form-configurator.ts

```diff
@@ -0,0 +1,414 @@
+import { LitElement, html, css } from 'lit';
+import { property, customElement } from 'lit/decorators.js';
+import { ChaiForm } from './chai-form';
+
+@customElement('chai-form-configurator')
+export class ChaiFormConfigurator extends LitElement {
+  @property({ type: String }) targetForm: string = '';
+
+  static override styles = css`
+   
+    .form-configurator-container
+    {
+      color: black;
+    padding: 20px;
+    background-color: #fbf7f1;
+    border-radius: 20px;
+    text-shadow: none;
+    font-weight: bolder;
+    font-size: 14px;
+    }
+
+    .row
+    {
+      margin-bottom:15px;
+      font-family:sans-serif;
+    }
+    .form-configurator-header-message
+    {
+      margin-bottom: 34px;
+    font-weight: bold;
+    }
+  `;
+
+  constructor() {
+    super();
+    this.targetForm = ''; 
+  }
+
+  @property({ type: String }) selectedVisibilityValue: string = '';
+
+
+  toggleChaiFormSlots(event : Event ,  componentName : string)
+  {
+
+    
+     const target = document.getElementById(this.targetForm) as ChaiForm; 
+        if (!target) return;
+
+            
+
+          const checkbox = event.target as HTMLInputElement; 
+        if(!checkbox) return;
+
+        if(componentName == 'chai-name')
+        {
+          target.isCreateName = checkbox.checked;
+        }else if(componentName == 'chai-email')
+        {
+          target.isCreateEmail = checkbox.checked;
+        }else if(componentName == 'chai-address')
+        {
+          target.isCreateAddress = checkbox.checked;
+        }else if(componentName == 'chai-phone')
+        {
+          target.isCreatePhone = checkbox.checked;
+        }
+  }
+
+
+  applyChaiStyle(event: Event , cssVaraible:string , px:string)
+  {
+    console.log(event,cssVaraible);
+      const target = document.getElementById(this.targetForm) as ChaiForm; 
+        if (!target) return;
+        const input = event.target as HTMLInputElement;
+        target.style.setProperty(cssVaraible , input.value + px);
+        
+  }
+
+
+  override render() {
+    return html`
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+
+<div>
+ 
+ <div class="row form-configurator-container">
+  <div class="col-sm-12">
+     <h2 class="form-configurator-header-message">Form Configurator</h2>
```

Comment:
There is _no_ way to make sense of all the inputs in this massive form. Add some grouping, make it one column, use an accordion control to group different settings together logically.
https://getbootstrap.com/docs/5.3/components/accordion/

Reply:
In addition, once it's one column, you can have various sub-columns in each accordion group: the simplest example would be one column for the label and one column for the input for that label. That way there's a clearer rhythm. But take a look at how Figma organizes its controls, it's very deliberate for each group of properties and it makes it easier to scan and find the section of controls that the user is looking for.